src/app.js

```diff
@@ -1,21 +1,31 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const fs = require('fs/promises');
 
-module.exports = sum;
+let correctAddress;
+const address = process.argv[3];
+
+fs.stat(address)
+  .then(stats => {
+    if (stats.isDirectory()) {
+      correctAddress = (!address.includes('/'))
+        ? `./${address}/${process.argv[2]}`
+        : `${address}${process.argv[2]}`;
+    }
+  })
+  .catch(error => {
+    if (error.code === 'ENOENT') {
+      if (!address.includes('/') || address[address.length - 1] !== '/') {
+        correctAddress = address;
+      } else {
+        console.log(error);
```

Comment:
better to throw an error - in this case you will be sure than next code will not be executed 
and you have already write catch for this error at the end